docker-compose/README.md

```diff
@@ -37,6 +37,31 @@ To enable access to these ports from outside the Docker network, you'll need to
 ## Docker Images
 Stargate Docker images are publicly available on [Docker Hub](https://hub.docker.com/r/stargateio/). To learn how the images are built or to build your own local versions, see the [coordinator node developer guide](../DEV_GUIDE.md) or API Service developer guides (under the [apis](apis) directory).
 
+The fastest way to build your own local images involves the following steps:
+
+* Local build of coordinator images
+  * Make sure you are in the repository root directory and have `JAVA_HOME` set to point to a JDK 1.8 installation
+  * Do a local build of jar files for coordinator:
+    ```
+    ./mvnw clean install -DskipTests
```

Comment:
missing `-P dse`, otherwise the dse will not be built, and the image building in the next step will not contain needed jars, which would make image not startable

Reply:
good catch, thanks